poker-pals/src/App.vue

```diff
@@ -5,24 +5,10 @@
 </template>
 
 <script>
-///// Uncomment the following lines if you want to use them as components in this file /////
-// import Login from './components/Login.vue';
-//import Poker from './components/Poker.vue';
-// import Profile from './components/Profile.vue';
```

Comment:
I think we still have this file. Might also want to remove that

Reply:
And `Header.vue`

Reply:
Fixed